pkg/utils/transport.go

```diff
@@ -12,8 +12,8 @@ import (
 
 // TODO move all this to library-go
 
-// transportFor returns an http.Transport for the given ca and client cert data (which may be empty)
-func transportFor(serverName string, caData, certData, keyData []byte) (http.RoundTripper, error) {
```

Comment:
is this even used by more than one package?

Reply:
Yes. I found multiple instances for it in `pkg/operator` and `pkg/controller`